container/docker/handler.go

```diff
@@ -184,6 +182,10 @@ func (self *dockerContainerHandler) GetSpec() (info.ContainerSpec, error) {
 	}
 
 	spec := libcontainerConfigToContainerSpec(libcontainerConfig, mi)
+
+	ctnr, _ := self.client.InspectContainer(self.id)
+	spec.HasNetwork = runconfig.NetworkMode(ctnr.HostConfig.NetworkMode).IsBridge()
```

Comment:
We can also check the [type](https://github.com/docker/libcontainer/blob/master/configs/network.go#L9) on the Network config.